projects/elife.yaml

```diff
@@ -292,24 +277,12 @@ builder:
 basebox:
```

Comment:
`basebox` is the last one to go; as long as there were projects on 14.04 we needed to be able to generate a AMI for them. [Current automation](https://github.com/elifesciences/elife-alfred-formula/blob/master/jenkinsfiles/Jenkinsfile.basebox-1804) targets an `aws-alt` configuration though.

Reply:
this is helpful, thank you.

either Graham will figure out how to decouple Lens and elife-api--legacy, or I'll finish upgrading that project to 18.04 and fold it into the stable of maintained projects. I'd prefer the former.

Reply:
Graham got Lens figured out.